hm_pyhelper/tests/test_lock_singleton.py

```diff
@@ -69,32 +72,34 @@ def fast_task():
         fast_thread.start()
 
     def test_lock_singleton_failure(self):
-
+        """
+        Start a slow running thread and then try to acquire a shared
+        lock. Expect the acquire call to throw an exception.
+        """
         lock = LockSingleton('test')
```

Comment:
@marvinmarnold I want to set the resource_count explicitly here. 
`lock = LockSingleton('test', resource_count=1)`